src/main/groovy/nebula/test/functional/GradleRunner.groovy

```diff
@@ -14,11 +14,46 @@
  * limitations under the License.
  */
 
-package nebula.test.functional;
+package nebula.test.functional
 
-import nebula.test.functional.internal.GradleHandle;
+import com.google.common.base.Predicate
+import com.google.common.base.Predicates
+import com.google.common.base.StandardSystemProperty
+import nebula.test.functional.internal.GradleHandle
 
 public interface GradleRunner {
+    // These predicates are here, instead of on GradleRunnerFactory due to a Groovy static compiler bug
```

Comment:
Known issue, fixed in a later version of Groovy - added a comment to that effect.